nixos/modules/services/mail/postfix.nix

```
@@ -584,6 +514,163 @@ in
               ])
             );
           options = {
+            message_size_limit = mkOption {
+              type = with types; nullOr int;
+              default = 10240000;
+              example = 52428800; # 50 MiB
+              description = ''
+                Maximum size of an email message in bytes.
+
+                <https://www.postfix.org/postconf.5.html#message_size_limit>
+              '';
+            };
+
+            mydestination = mkOption {
+              type =
+                with types;
+                nullOr (oneOf [
+                  str
+                  (listOf str)
```

```suggestion
                oneOf [
                  null
                  str
                  (listOf str)
```
work work if `types.null` were a thing. Should it?

---

It's not